.github/workflows/quarkus.yml

```diff
@@ -0,0 +1,371 @@
+          -d postgres:10.5
+        if: matrix.postgres
+      - name: MySQL Service
+        run: |
+          sudo service mysql stop || true
+          docker run --rm --publish 3306:3306 --name build-mysql  \
+            -e MYSQL_USER=$DB_USER -e MYSQL_PASSWORD=$DB_PASSWORD -e MYSQL_DATABASE=$DB_NAME -e MYSQL_RANDOM_ROOT_PASSWORD=true \
+            -d mysql:5 --skip-ssl
+        if: matrix.mysql
+      - name: DB2 Service
+        run: |
+          docker run --rm --publish 50000:50000 --name build-db2 --privileged=true \
+            -e DB2INSTANCE=hreact -e DB2INST1_PASSWORD=hreact -e DBNAME=hreact -e LICENSE=accept -e AUTOCONFIG=false -e ARCHIVE_LOGS=false \
+            -d ibmcom/db2:11.5.0.0a
+        if: matrix.db2
+      - name: Maria DB Service
+        run: |
+          docker run --rm --publish 3308:3306 --name build-mariadb \
+            -e MYSQL_USER=$DB_USER -e MYSQL_PASSWORD=$DB_PASSWORD -e MYSQL_DATABASE=$DB_NAME -e MYSQL_ROOT_PASSWORD=secret \
+            -d mariadb:10.4
+        if: matrix.mariadb
+      - name: MS-SQL Service
+        run: |
+          docker run --rm --publish 1433:1433 --name build-mssql \
+            -e ACCEPT_EULA=Y -e SA_PASSWORD=ActuallyRequired11Complexity \
+            -d microsoft/mssql-server-linux:2017-CU13
+        if: matrix.mssql
+      - name: Amazon Services
+        run: |
+          docker run --rm --publish 8000:4569 --publish 8008:4572 --publish 8009:4575 --publish 8010:4576 --publish 8011:4599 --publish 8012:4566 --name build-amazon-service-clients -e SERVICES=s3,dynamodb,sns,sqs,kms,ses -e START_WEB=0 \
+            -d localstack/localstack:0.11.1
+        if: matrix.amazonServices
+      - name: Neo4j Service
+        run: |
+          docker run --rm --publish 7687:7687 --name build-neo4j \
+            -e NEO4J_AUTH=neo4j/secret -e NEO4J_dbms_memory_pagecache_size=10M -e NEO4J_dbms_memory_heap_initial__size=10M \
+            -d neo4j/neo4j-experimental:4.0.0-rc01
+        if: matrix.neo4j
+      - name: Redis Service
+        run: docker run --rm --publish 6379:6379 --name build-redis -d redis:5.0.8-alpine
+        if: matrix.redis
+      - name: Keycloak Service
+        run: |
+          docker run --rm --publish 8180:8080 --publish 8543:8443 --name build-keycloak \
+            -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin -e JAVA_OPTS=" \
+              -server -Xms64m -Xmx512m -XX:MetaspaceSize=96M \
+              -XX:MaxMetaspaceSize=256m -Djava.net.preferIPv4Stack=true -Djava.awt.headless=true \
+              -Dkeycloak.profile.feature.upload_scripts=enabled" \
+            -d quay.io/keycloak/keycloak:11.0.0
+        if: matrix.keycloak
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        # Uses sha for added security since tags can be updated
+        uses: joschi/setup-jdk@b9cc6eabf7e7e3889766b5cee486f874c9e1bd2d
+        with:
+          java-version: 11
+      - name: Reclaim Disk Space
+        run: .github/ci-prerequisites.sh
```

Comment:
@zakkak, please add `.github/ci-prerequisites.sh` or remove this step.

```
Run .github/ci-prerequisites.sh
/home/runner/work/_temp/cfae83e8-7803-4c08-9ef8-59de516c2f55.sh: line 1: .github/ci-prerequisites.sh: No such file or directory
##[error]Process completed with exit code 127.
```
https://github.com/zakkak/mandrel/runs/993279323?check_suite_focus=true